src/api/graphql/resolver_handler/Session.ts

```diff
@@ -0,0 +1,134 @@
+import { Mutex } from 'async-mutex';
+import Logger from '../../../common/logger';
+import * as constants from '../../../common/constants';
+import getDatabase from '../../../database';
+import * as SessionTypes from '../types/Session';
+
+const Session = getDatabase().getSession();
+const db = getDatabase().getInstance();
+const log = Logger.createLogger('graphql.resolver_handler.Session');
+const mutex = new Mutex();
+
+const SCHEDULE_TEMPLATE = {
+  create: 'CREATE EVENT {evant_name} ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 2 DAYS DO UPDATE somul.Session SET `admin_approved` = "2" WHERE (`library_id` = "{library_id}" AND `user_email` = "{user_email}");',
```

Comment:
evant_name 오타 인 것 같습니다!
```suggestion
  create: 'CREATE EVENT {event_name} ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 2 DAYS DO UPDATE somul.Session SET `admin_approved` = "2" WHERE (`library_id` = "{library_id}" AND `user_email` = "{user_email}");',
```

Reply:
반영하겠습니다!